website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -184,11 +185,85 @@ then you have to manually add more EC2NodeClasses and point their NodePools to t
+
+* v0.37.1
+* v0.36.3
+* v0.35.6
+* v0.34.7
+* v0.33.6
+
+{{% alert title="Note" color="warning" %}}
+When rolling back from v1, Karpenter will not retain data that was only valid in v1 APIs. For instance, if you were upgrading from v0.33.5 to v1, updated the `NodePool.Spec.Disruption.Budgets` field and then rolled back to v0.33.6, Karpenter would not retain the `NodePool.Spec.Disruption.Budgets` field, as that was introduced in v0.34.x. If you are configuring the kubelet field, and have removed the `compatibility.karpenter.sh/v1beta1-kubelet-conversion` annotation, rollback is not supported without replacing your nodes between EC2NodeClass and NodePool.
+{{% /alert %}}
+
+{{% alert title="Note" color="warning" %}}
+Since both v1beta1 and v1 will be served, `kubectl` will default to returning the `v1` version of your CRDs. To interact with the v1beta1 version of your CRDs, you'll need to add the full resource path (including api version) into `kubectl` calls. For example: `k get nodeclaim.v1beta1.karpenter.sh`
+{{% /alert %}}
+
+1. Set environment variables
+
+```bash
+export KARPENTER_NAMESPACE="kube-system"
+export KARPENTER_VERSION="<rollback version of karpenter>"
+export CLUSTER_NAME="<name of your cluster>"
+export TEMPOUT="$(mktemp)"
+```
+
+{{% alert title="Warning" color="warning" %}}
+If you open a new shell to run steps in this procedure, you need to set some or all of the environment variables again.
+To remind yourself of these values, type:
+
+```bash
+echo "${KARPENTER_NAMESPACE}" "${KARPENTER_VERSION}" "${CLUSTER_NAME}" "${TEMPOUT}"
+```
+
+{{% /alert %}}
+
+2. Rollback the Karpenter Policy
+
+```bash
+curl -fsSL https://raw.githubusercontent.com/aws/karpenter-provider-aws/v"${KARPENTER_VERSION}"/website/content/en/docs/getting-started/getting-started-with-karpenter/cloudformation.yaml > ${TEMPOUT} \
+    && aws cloudformation deploy \
+    --stack-name "Karpenter-${CLUSTER_NAME}" \
+    --template-file "${TEMPOUT}" \
+    --capabilities CAPABILITY_NAMED_IAM \
+    --parameter-overrides "ClusterName=${CLUSTER_NAME}"
+```
+
+3. Rollback the CRDs
+
+```bash
+helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd --version "${KARPENTER_VERSION}" --namespace "${KARPENTER_NAMESPACE}" --create-namespace \
+  --set webhook.enabled=true \
+  --set webhook.serviceName=karpenter \
+  --set webhook.serviceNamespace="${KARPENTER_NAMESPACE}" \
+  --set webhook.port=8443
+```
+
+4. Rollback the Karpenter Controller 
+
+```bash
+helm upgrade --install karpenter oci://public.ecr.aws/karpenter/karpenter --version ${KARPENTER_VERSION} --namespace "${KARPENTER_NAMESPACE}" --create-namespace \
+  --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"=${KARPENTER_IAM_ROLE_ARN} \
```

Comment:
We have just been doing the EKS pod identity version of this, I think we are good to drop this line since we don't have it in other installation calls earlier in this file

Reply:
I was thinking of leaving the service account flag, since it handles both IRSA and pod identity. If the user has pod identity that will be prioritized. User will just loss the ability to use IRSA from our helm command